sal_interfaces/MTAOS/MTAOS_Events.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>FilterType_u,FilterType_g,FilterType_r,FilterType_i,FilterType_z,FilterType_y</Enumeration>
```

Comment:
This is inconsistent with the IDL now:
https://github.com/lsst-ts/ts_idl/pull/57/commits/1b6c0edd9deabd9cd9cb1091d464bf6bdff177fa

Reply:
updated in both places.